PowerShellEditorServices.build.ps1

```diff
@@ -372,6 +376,21 @@ task TestHost {
     exec { & $script:dotnetExe test -f $script:TestRuntime.Core (DotNetTestFilter) }
 }
 
+task TestPester Build,BuildPsesClientModule,EnsurePesterInstalled,{
+    $pwshExe = (Get-Process -Id $PID).Path
+    $pesterTestDir = Resolve-Path "$PSScriptRoot/test/Pester/"
+    exec { & $pwshExe -Command "cd $pesterTestDir; Invoke-Pester" }
```

Comment:
Is there a reason you're starting a new process instead of just Invoke-Pester in the current process?

Reply:
I switched it over and now no build hang!

Had it like this to isolate the module loading, but I think in-proc makes more sense. Still mysterious to me why `exec` doesn't clean this up.